specfile/specfile.py

```diff
@@ -50,19 +65,25 @@ class Specfile:
         autosave: Whether to automatically save any changes made.
     """
 
+    ENCODING_ARGS: EncodingArgs = {"encoding": "utf8", "errors": "surrogateescape"}
+
     def __init__(
         self,
-        path: Union[Path, str],
+        path: Optional[Union[Path, str]] = None,
+        content: Optional[str] = None,
+        file: Optional[IO] = None,
         sourcedir: Optional[Union[Path, str]] = None,
         autosave: bool = False,
         macros: Optional[List[Tuple[str, Optional[str]]]] = None,
         force_parse: bool = False,
     ) -> None:
         """
         Initializes a specfile object.
-
+        The arguments `path`, `file` and `content` are mutually exclusive and only one can be used.
```

Comment:
I'm not able to apply the suggestion from the start of the docstring, nevertheless:

```suggestion
        Initializes a specfile object. You can specify either a path to the spec file,
        its content as a string or a file object representing it. `sourcedir` is optional
        if `path` or a named `file` is provided and will be set to the parent directory.
        
```